src/oge/validation.py

```diff
@@ -18,27 +22,27 @@
 
 
 def validate_year(year):
-    """Returns a warning if the year specified is not known to work with the pipeline."""
-
-    if year < earliest_validated_year:
-        year_warning = f"""
-        ################################################################################
-        The data pipeline has only been validated to work for years {earliest_validated_year}-{latest_validated_year}.
-        Running the pipeline for {year} may cause it to fail or may lead to poor-quality
-        or anomalous results. To check on the progress of validating additional years of
-        data, see: https://github.com/singularity-energy/open-grid-emissions/issues/117
-        ################################################################################
-        """
-        logger.warning(year_warning)
-    elif year > latest_validated_year:
-        year_warning = f"""
-        ################################################################################
-        The most recent available year of input data is currently {latest_validated_year}.
-        Input data for {year} should be available from the EIA in Fall {year+1} and we will
-        work to validate that the pipeline works with {year} data as soon as possible
-        after the data is released.
-        ################################################################################
-        """
+    """Raises a warning if the year specified is not known to work with the pipeline.
+
+    Args:
+        year (_type_): a four-digit year.
+
+    Raises:
+        UserWarning: if `year` is not supported.
+    """
+    start = earliest_data_year
```

Comment:
Not sure if we need to rename these or if we can just use these directly, but doesn't really matter that much